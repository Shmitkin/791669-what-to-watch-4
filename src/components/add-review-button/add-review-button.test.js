import React from "react";
import renderer from "react-test-renderer";
import {BrowserRouter} from "react-router-dom";

import AddReviewButton from "./add-review-button.jsx";

it(`Should AddReviewButton render correctly`, () => {
  const tree = renderer
    .create(
        <BrowserRouter>
          <AddReviewButton movieId={`id`}/>
        </BrowserRouter>
    )
    .toJSON();

  expect(tree).toMatchSnapshot();
});
