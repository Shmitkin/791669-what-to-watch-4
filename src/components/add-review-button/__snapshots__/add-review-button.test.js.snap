// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should AddReviewButton render correctly 1`] = `
<a
  className="btn movie-card__button"
  href="/films/id/review"
  onClick={[Function]}
>
  Add review
</a>
`;
